postmaster/static/js/utils.js

```diff
@@ -51,20 +51,25 @@ function changePage(obj, e) {
 // This manages the loading spinner for the table
 function manageSpinner(present) {
 
-    var spinner = $('div.loader');
+    var spinner = $('div.spinner');
+    var spinnerParent = $('div.spinnerParent');
 
     if (present) {
-        if (spinner.length == 0) {
-            $('#dynamicTableDiv').prepend('<div class="loader"></div>');
+        // If the spinner isn't there, create it
+        if (spinnerParent.length == 0) {
+            var spinnerParentHtml = $('<div/>', {'class': 'spinnerParent'});
+            var spinnerHtml = $('<div/>', {'class': 'spinner'});
+            spinnerHtml.append($('<div/>', {'class': 'bounce1'}));
+            spinnerHtml.append($('<div/>', {'class': 'bounce2'}));
+            spinnerHtml.append($('<div/>', {'class': 'bounce3'}));
```

Comment:
You prefer the following?
```
spinnerHtml.append($('<div/>').addClass('bounce1'));
```

It's really the same thing except this other way you create the div and then add the class, instead of creating the div with the class.